transforms/.make.cicd.targets

```diff
@@ -7,10 +7,15 @@
 include $(REPOROOT)/transforms/.make.transforms
 
 ######################################################################
-## Default setting for TRANSFORM_RUNTIME uses folder name-- Old layout
-TRANSFORM_PYTHON_RUNTIME_SRC_FILE=-m dpk_$(TRANSFORM_NAME).transform
-TRANSFORM_RAY_RUNTIME_SRC_FILE=-m dpk_$(TRANSFORM_NAME).ray.transform
-TRANSFORM_PYTHON_RUNTIME_SRC_FILE=-m dpk_$(TRANSFORM_NAME).spark.transform
+## Default setting for TRANSFORM_RUNTIME entry point: 
+# python -m dpk_html2parquet.ray.transform --help
+# or
+# python -m dpk_html2parquet.transform_python --help
+#
+TRANSFORM_PYTHON_SRC?="-m dpk_$(TRANSFORM_NAME).transform_python"
```

Comment:
I guess there's a reason for not inclucing .py in the value?  SRC in the var name sort of suggests (to me) that it is the name of a source file.  Actually, this looks like a package name.  Does changing _SRC to _PKG make sense for these?

Reply:
I don't think we need to do this. _SRC is fine. again, I will be taking another pass at all the Makefile once we get all the current work in place.